web/src/components/CanvasBoard.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from './CanvasBoard.module.css';
+
+type Props = {
+  height: number,
+  width: number,
+}
+
+const CanvasBoard = ({ height, width }: Props) => {
```

Comment:
assigning the react code to a variable feels like we're implying a component might export multiple react fragments.

Best to return without assigning imo